Assets/Scripts/Controllers/InputOutput/BuildModeController.cs

```diff
@@ -275,9 +275,6 @@ public void DoBuild(Tile tile)
         }
         else if (buildMode == BuildMode.FLOOR)
         {
-            // We are in tile-changing mode.
-            ////t.Type = buildModeTile;
```

Comment:
If this code is no longer used, it should be removed entirely rather than commented out. And in general, unless there is a very good reason, please try to avoid the stylecop comment checks with the quad slash.